src/tools/race/CMakeLists.txt

```diff
@@ -1,14 +1,20 @@
 find_package(Threads)
+find_package(LibRt)
 
 if (APPLE)
 	remove_tool(race "race tool does not compile under MAC OS X")
 else()
+	if (NOT LIBRT_FOUND)
+		remove_tool(race "race tool requires librt support")
```

Comment:
> Is it possible that librt is not found but nevertheless available?

I hope not.

>  Why doing something different than for the resolver here?

My assumption was that the resolver compiles without librt. Is that correct or do we remove the resolver if no librt is found?

The race tool seems to require librt, so removing it in case of a missing librt was the only way.